resources/sdk/purecloudjava/templates/ApiClient.mustache

```diff
@@ -975,4 +985,61 @@ public class ApiClient implements AutoCloseable {
         @Override
         public void close() throws Exception { }
     }
+
+     private ApiClientConnectorResponse retryInvoke(ApiClientConnectorRequest connectorRequest, ApiClientConnectorResponse connectorResponse) {
+         int retriesBeforeBackOff = 0;
```

Comment:
Is this the retries before a fixed backoff bucket? Consider tweaking the name a little bit for clarity